code/src/components/Button.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+export const Button = ({onClick, text}) => (
+    <button onClick={onClick}>
+        {text}
+    </button>

```

Comment:
Super cool that you passed the onClick function and the text to the buttons as well so they're very re-usable! I really should be using more props on my own projects I can see